src/test/java/org/jabref/preferences/JabRefPreferencesTest.java

```diff
@@ -0,0 +1,26 @@
+package org.jabref.preferences;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class JabRefPreferencesTest {
+    private static Stream<Arguments> provideTestData() {
+        return Stream.of(
+                Arguments.of(List.of("A", "B", "C", "D"), "A;B;C;D"),
+                Arguments.of(List.of("A", "B", "C", ""), "A;B;C;")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideTestData")
+    void roundTrip(List<String> sampleList, String sampleString) {
+        assertEquals(sampleString, JabRefPreferences.convertListToString(sampleList));
+        assertEquals(sampleList, JabRefPreferences.convertStringToList(sampleString));
+    }
```

Comment:
I would split these up into two methods - they can use the same `MethodSource`. The two assertions are independent of each other. The conversion result is not stored in `sampleString` used in line 24.

Reply:
done